### PR TITLE
Add option to set target of Export html:a button.

### PR DESCRIPTION
**Pull request by hlavacz**
I need it to open new tab to create pdf of filtered rows.

### CODE REVIEW

File: src/Export/Export.php [hunk 1]
```diff
@@ -58,6 +58,12 @@ class Export
 	protected $confirmDialog = null;
 
 
+	/**
+	 * @var string
+	 */
+	protected $target = null;
```
Comment (paveljanda) [inline src/Export/Export.php:64]: Could you please change that to `@var string|null`? Thx

**Review by paveljanda — CHANGES_REQUESTED** [review]

File: src/Export/Export.php (hunk 1 above)
Comment (hlavacz) [reply src/Export/Export.php:64]: My bad, sorry. It is fixed now.

**Comment by paveljanda** [comment]
Could you please fix travis? https://travis-ci.org/github/contributte/datagrid/jobs/725164052
thx

**Comment by hlavacz** [comment]
Travis fixed.

**Review by paveljanda — APPROVED** [review]

**Comment by paveljanda** [comment]
Thanks! :)

**Comment by paveljanda** [comment]
@hlavacz Uuh, one more thing - could you add that pls to docs? :) https://github.com/contributte/datagrid/tree/master/.docs

**Comment by hlavacz** [comment]
Sorry, I am not familier with github as I should be... I pushed commit of doc update on mine fork but I am afraid you can't see it after you aprooved this pull request right? So I need to create a new pull request to commit of doc file? I cant see any other option..

**Comment by paveljanda** [comment]
Yes, a new pull request is needed. If you won't be able to work it out, you can send it to me here and I will edit the master branch manually, np.

**Comment by hlavacz** [comment]
Edit manualy Export.md is probebly easest for both, Few words only.. :) Line 25

``You can tell whether to use ajax or not (`->setAjax()`). Or a button title (`->setTitle('Title')`). Or target (`->setTarget('_blank')`).``

**Comment by paveljanda** [comment]
Thanks 👍 

https://github.com/contributte/datagrid/releases/tag/v6.2.28

**Pull request merged**